README.md

```diff
@@ -398,5 +399,26 @@ nav:
   search: /search/
 ```
 
+### Custom Github 404 page
+
+When you hosting your hexo blog use Github Page, if Visitors a not exist post page, it will be redicrt to a deafult github 404 page. about this , you can custom the 404 page.
```

Comment:
I am not sure whether my assumptions are all correct, but I'll give it a shot.

Should be changed to
```markdown
If visitors go to a non-existing page on your hexo blog, they usually would be redirected to the default Github 404 page, if the blog is hosted on Github.
In order to customize the 404 page, follow the next steps.
```